<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConcentrationImage</class>
 <widget class="QDialog" name="DialogConcentrationImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Initial Concentration Image</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="grpImage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabelMouseTracker" name="lblImage" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>&lt;h4&gt;Initial concentration&lt;/h4&gt;
&lt;p&gt;An image of the initial concentration resulting from the supplied analytic expression.&lt;/p&gt;</string>
         </property>
         <property name="text" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="grpButtons">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QLineEdit" name="txtMinConc"/>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="lblMaxConcUnits">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>units</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QLabel" name="lblMaxConcColour">
         <property name="text">
          <string>white</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="lblMaxConcLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Maximum concentration</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="15" column="0" colspan="2">
        <widget class="QPushButton" name="btnSmoothImage">
         <property name="text">
          <string>smooth image</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="Line" name="lineMax">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="lblMinConcColour">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>black</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="2">
        <widget class="QPushButton" name="btnImportImage">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Import an image of the spatial concentration of this species</string>
         </property>
         <property name="statusTip">
          <string>Import an image of the spatial concentration of this species</string>
         </property>
         <property name="whatsThis">
          <string>&lt;h4&gt;import from image&lt;/h4&gt;&lt;p&gt;Import an image of the spatial concentration of this species&lt;/p&gt;</string>
         </property>
         <property name="text">
          <string>import image...</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0" colspan="2">
        <widget class="QLabel" name="lblConcentration">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLineEdit" name="txtMaxConc"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="Line" name="lineMin">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="lblMinConcLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Mininum concentration</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="lblMinConcUnits">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>units</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0" colspan="2">
        <widget class="QComboBox" name="cmbExampleImages">
         <property name="currentText">
          <string>load example image...</string>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>load example image...</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>one-blob-100x100</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>two-blobs-100x100</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>three-blobs-100x100</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gradient-100x100</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QPushButton" name="btnExportImage">
         <property name="toolTip">
          <string>Export an image of the spatial concentration of this species</string>
         </property>
         <property name="statusTip">
          <string>Export an image of the spatial concentration of this species</string>
         </property>
         <property name="whatsThis">
          <string>Export an image of the spatial concentration of this species</string>
         </property>
         <property name="text">
          <string>export image...</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="chkGrid">
         <property name="text">
          <string>Show &amp;grid</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QCheckBox" name="chkScale">
         <property name="text">
          <string>Show &amp;scale</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLabelMouseTracker</class>
   <extends>QWidget</extends>
   <header>qlabelmousetracker.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtMinConc</tabstop>
  <tabstop>txtMaxConc</tabstop>
  <tabstop>chkGrid</tabstop>
  <tabstop>chkScale</tabstop>
  <tabstop>btnImportImage</tabstop>
  <tabstop>btnExportImage</tabstop>
  <tabstop>cmbExampleImages</tabstop>
  <tabstop>btnSmoothImage</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
